apps/sentry/src/connectors/users.ts

```diff
@@ -1,27 +1,69 @@
-/**
- * DISCLAIMER:
- * This is an example connector, the function has a poor implementation. When requesting against API endpoint we might prefer
- * to valid the response data received using zod than unsafely assign types to it.
- * This might not fit your usecase if you are using a SDK to connect to the Saas.
- * These file illustrate potential scenarios and methodologies relevant for SaaS integration.
- */
-
-import { MySaasError } from './commons/error';
-
-export type MySaasUser = {
-  id: string;
-  username: string;
-  email: string;
+import { env } from '@/env';
+import { SentryError } from './commons/error';
+
+export type SentryUser = {
+  role: string;
+  user: {
+    id: string;
```

Comment:
We have also access to the field "has2fa" in api. we need to access that too.

Reply:
Done